resources/fixtures/Showtec/Showtec-Acrobat.qxf

```diff
@@ -18,51 +18,51 @@
-  <Capability Color2="#ffaa00" Min="112" Max="119" Color="#00ffff">Cyan + Orange</Capability>
-  <Capability Color2="#ffffff" Min="120" Max="127" Color="#ffaa00">Orange + White</Capability>
-  <Capability Res="Others/rainbow.png" Min="128" Max="191">Forwards (CW) Rainbow, Slow - Fast</Capability>
-  <Capability Res="Others/rainbow.png" Min="192" Max="255">Bacwards (CCW) Rainbow, Slow - Fast</Capability>
+  <Capability Min="0" Max="7" Preset="ColorMacro" Res1="#ffffff">Open / White</Capability>
+  <Capability Min="8" Max="15" Preset="ColorMacro" Res1="#0000ff">Blue</Capability>
+  <Capability Min="16" Max="23" Preset="ColorMacro" Res1="#ffff00">Yellow</Capability>
+  <Capability Min="24" Max="31" Preset="ColorMacro" Res1="#ff072c">Peachblow</Capability>
+  <Capability Min="32" Max="39" Preset="ColorMacro" Res1="#00ff00">Green</Capability>
+  <Capability Min="40" Max="47" Preset="ColorMacro" Res1="#ff14f4">Pink</Capability>
+  <Capability Min="48" Max="55" Preset="ColorMacro" Res1="#00ffff">Cyan</Capability>
+  <Capability Min="56" Max="63" Preset="ColorMacro" Res1="#ffaa00">Orange</Capability>
+  <Capability Min="64" Max="71" Preset="ColorDoubleMacro" Res1="#ffffff" Res2="#0000ff">White + Blue</Capability>
+  <Capability Min="72" Max="79" Preset="ColorDoubleMacro" Res1="#0000ff" Res2="#ffff00">Blue + Yellow</Capability>
+  <Capability Min="80" Max="87" Preset="ColorDoubleMacro" Res1="#ffff00" Res2="#ff0533">Yellow + Peachblow</Capability>
+  <Capability Min="88" Max="95" Preset="ColorDoubleMacro" Res1="#fc072c" Res2="#00ff00">Peachblow + Green</Capability>
+  <Capability Min="96" Max="103" Preset="ColorDoubleMacro" Res1="#00ff00" Res2="#ff06fb">Green + Pink</Capability>
+  <Capability Min="104" Max="111" Preset="ColorDoubleMacro" Res1="#ff03fb" Res2="#00ffff">Pink + Cyan</Capability>
+  <Capability Min="112" Max="119" Preset="ColorDoubleMacro" Res1="#00ffff" Res2="#ffaa00">Cyan + Orange</Capability>
+  <Capability Min="120" Max="127" Preset="ColorDoubleMacro" Res1="#ffaa00" Res2="#ffffff">Orange + White</Capability>
+  <Capability Min="128" Max="191" Preset="GoboMacro" Res1="Others/rainbow.png">Forwards (CW) Rainbow, Slow - Fast</Capability>
+  <Capability Min="192" Max="255" Preset="GoboMacro" Res1="Others/rainbow.png">Bacwards (CCW) Rainbow, Slow - Fast</Capability>
  </Channel>
  <Channel Name="Shutter / Strobe">
   <Group Byte="0">Shutter</Group>
-  <Capability Min="0" Max="3">Closed / Blackout</Capability>
-  <Capability Min="4" Max="7">Shutter Open</Capability>
-  <Capability Min="8" Max="215">Strobe, Slow - Fast ( 0-10 Flash / Sec )</Capability>
-  <Capability Min="216" Max="255">Shutter Open</Capability>
+  <Capability Min="0" Max="3" Preset="ShutterClose">Closed / Blackout</Capability>
+  <Capability Min="4" Max="7" Preset="ShutterOpen">Shutter Open</Capability>
+  <Capability Min="8" Max="215" Preset="StrobeSlowToFast">Strobe, Slow - Fast ( 0-10 Flash / Sec )</Capability>
```

Comment:
I usually set it as StrobeFreqRange 0-10

Reply:
Ok, should I change it?